app/views/advanced/_advanced_search_help.html.erb

```diff
@@ -1,26 +1,20 @@
+<%# OVERRIDE Blacklight Advanced Search v7.0.0 to add Search by date %>
+
 <div class='card card-default'>
   <div class="card-body">
     <h4 class="card-title">Search tips</h4>
     <ul class="advanced-help">
-     <li>Select "match all" to require all fields.
-     </li>
-
-     <li>Select "match any" to find at least one field.
-     </li>
-
-     <li>Combine keywords and attributes to find specific items.
-     </li>
-
-     <li>Search by date with format: YYYYMMDD, YYYYMM or YYYY. Do not use "-", "/", or any other special characters.</li>
-
-     <li>Use quotation marks to search as a phrase.
-
-     <li>Use "+" before a term to make it required. (Otherwise results matching only some of your terms may be included).</li>
-
-     <li>Use "-" before a word or phrase to exclude.
-
-     <li>Use "OR", "AND", and "NOT" to create complex boolean logic. You can use parentheses in your complex expressions. </li>
-     <li>Truncation and wildcards are not supported - word-stemming is done automatically.</li>
-   </ul>
+      <li>Select "match all" to require all fields.</li>
+      <li>Select "match any" to find at least one field.</li>
+      <li>Combine keywords and attributes to find specific items.</li>
+      <%# OVERRIDE begin %>
```

Comment:
Why is this a BEGIN override?

Reply:
because we're adding
```erb
<li>Search by date with format: YYYYMMDD, YYYYMM or YYYY. Do not use "-", "/", or any other special characters.</li>
```
which Blacklight Advanced Search didn't seem to have

https://github.com/projectblacklight/blacklight/blob/v7.35.0/app/views/catalog/_advanced_search_help.html.erb